client/account-recovery/account-recovery-root/index.jsx

```diff
@@ -35,13 +39,35 @@ const getPageInfo = ( translate, step ) => {
 			trackerTitle: 'Account Recovery > Reset Password',
 			documentHeadTitle: translate( 'Reset Password ‹ Account Recovery' ),
 		},
-
+		[ STEPS.RESET_PASSWORD_EMAIL ]: {
+			trackerTitle: 'Account Recovery > Reset Password Email',
+			documentHeadTitle: translate( 'Reset Password < Email' ),
```

Comment:
Thanks for fixing the edge case :)
I think it won't do harm to update it as the breadcrumb format, so I've updated it in 632ff580855198c42370d7eb8ade38646ebfea76.